optuna/integration/mxnet.py

```diff
@@ -9,8 +9,8 @@
 class MXNetPruningCallback(object):
     """MXNet callback to prune unpromising trials.
 
-    See `the example <https://github.com/optuna/optuna/blob/master/
-    examples/mxnet/mxnet_integration.py>`__
+    See `the example <https://github.com/optuna/optuna-examples/blob/main/
+    mxnet/mxnet_simple.py>`__
```

Comment:
`mxnet_simple.py` does not use `MXNetPruningCallback`. Please revert the filename.
```suggestion
    mxnet/mxnet_integration.py>`__
```